doc/installation/install_mac.dox

```diff
@@ -80,218 +94,260 @@ The following options are also available:
 --HEAD
  	Install HEAD version
 \endcode
- 
-After the installation, you will need to add the $YARP_DATA_DIRS environmental variable (in order to follow the KISS principle we are using the native mac editor Textedit. Please feel free to use whichever tool/editor you prefer):
- 
-\code
-cd ~
-open -a TextEdit .bash_profile
-\endcode
- 
-If the following command result in an error: The file .... does not exist, do the following first:
 
-\code
-cd ~
-touch .bash_profile
-\endcode
- 
-and add the line:
+After the installation, you will need to set your \subpage environment_configuration.
 
-\code
-export YARP_DATA_DIRS=/usr/local/share/yarp
-\endcode
- 
-To refresh and load these environment settings type:
 
-\code
-source ~/.bash_profile 
-\endcode
- 
-Now you can proceed with the following link: \subpage check_your_installation.
+\subsection install_mac_from_source Install YARP from source
+If the options above are not enough and you want to install YARP with additional options or in a difference location, you can compile it from source, directly from the repository. First install the dependencies, download the code from the repository and then proceed with the installation.
 
-\subsection install_mac_bash_completion_brew Bash completion
-Installing YARP using Homebrew, the bash completion is automatically installed and enabled.
+ \subsubsection install_mac_YARP_Dependencies Installing YARP Dependencies
+ Here we provide information for installing the <b>YARP dependencies</b> via Homebrew.
+ In order to install the YARP dependencies enter the following command in a terminal:
 
-\section install_mac_from_source Install YARP from source
-If the options above are not enough and you want to install YARP with additional options or in a difference location, you can compile it from source, directly from the repository.
- 
-\subsection install_mac_YARP_Dependencies Installing YARP Dependencies
-Here we provide information for installing the <b>YARP dependencies</b> via <b>Homebrew</b>
-In order to install the YARP dependencies enter the following command in a terminal:
+ \code
+ brew install yarp --only-dependencies
+ \endcode
 
-\code
-brew install yarp --only-dependencies
-\endcode
- 
 
-For compiling the required bindings (LUA, JAVA, PYTHON etc) you need <b>swig</b>
+ For compiling the required bindings (LUA, JAVA, PYTHON etc) you need <b>swig</b>
```

Comment:
Little style comment: Lua, Python, Java are not acronyms, and so typically are not capitalized.